docs/components/CONTAINER_PRESENTER.md

```diff
@@ -112,9 +72,7 @@ passengers/
 
 ## Guidelines summary
 
-* A component interacting with external entities (store, services, etc.) should be split into a container and a presenter
-* A component with no interactions with external entities should only be a presenter
-* By definition, a block is always split into container/presenter
+* A component interacting with external entities (store, services, etc.) may be split into a container and a presenter
 * Subcomponents are located in the related block subcomponents folder. If the subcomponent is a block, it will then be set either in the components root folder (if it has a functional meaning), or at the root of the functional folder in a dedicated sub folder.
```

Comment:
```suggestion
* Subcomponents are located in the subcomponents folder of the related block. If the subcomponent is a block, it will then be set either in the component's root folder (if it has a functional meaning), or at the root of the functional folder in a dedicated subfolder.
```